package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "thali-cordova-plugin-jxcore",
```

Comment:
Shouldn't the name just be 'thali'? That is what we will eventually release it all under when it gets posted to https://www.npmjs.com/package/thali